packages/fluentui/docs/src/examples/components/Tree/Usage/TreeMultiselectAsListExample.shorthand.tsx

```diff
@@ -0,0 +1,136 @@
+import * as React from 'react';
+import { Tree, Text, treeAsListBehavior } from '@fluentui/react-northstar';
+
+const items = [
+  {
+    id: 'tree-item-1',
+    title: 'House Lannister',
+    items: [
+      {
+        id: 'tree-item-11',
+        title: 'Tywin',
+        selectableParent: true,
+        items: [
+          {
+            id: '1',
+            title: 'Jaime',
+            items: [
+              {
+                id: '2',
+                title: 'Jaime 2',
+              },
+              {
+                id: '3',
+                title: 'Jaime 3',
+              },
+            ],
+          },
+          {
+            id: '4',
+            title: 'Cersei',
+            selectionIndicator: {
+              children: (Component, { selected, onClick, ...props }) => {
+                return (
+                  <Component {...props}>
+                    <input
+                      data-is-focusable={false}
+                      type="checkbox"
+                      checked={selected}
+                      onClick={onClick}
+                      onChange={() => {}}
+                    />
+                  </Component>
+                );
+              },
+            },
+          },
+          {
+            id: '5',
+            title: 'Tyrion',
+          },
+        ],
+      },
+      {
+        id: 'tree-item-12',
+        title: 'Kevan',
+        items: [
+          {
+            id: 'tree-item-121',
+            title: 'Lancel',
+          },
+          {
+            id: 'tree-item-122',
+            title: 'Willem',
+          },
+          {
+            id: 'tree-item-123',
+            title: 'Martyn',
+          },
+        ],
+      },
+    ],
+  },
+  {
+    id: 'tree-item-2',
+    title: 'House Targaryen',
+    selectionIndicator: {
+      children: (Component, { expanded, selected, ...props }) => {
+        return <Text {...props} content={selected ? 'unselect all' : 'select all'} />;
+      },
+    },
+    selectableParent: true,
+    items: [
+      {
+        id: 'tree-item-21',
+        title: 'Aerys',
+        items: [
+          {
+            id: 'tree-item-211',
+            title: 'Rhaegar',
+          },
+          {
+            id: 'tree-item-212',
+            title: 'Viserys',
+          },
+          {
+            id: 'tree-item-213',
+            title: 'Daenerys',
+            selectable: false,
+          },
+        ],
+      },
+    ],
+  },
+  {
+    id: '100',
+    title: 'House Skywalker',
+    selectableParent: true,
+    items: [
+      {
+        id: '102',
+        title: 'Leia',
+      },
+      {
+        id: '103',
+        title: 'Luke',
+      },
+      {
+        id: '104',
+        title: 'Rey',
+        selectable: false,
+      },
+    ],
+  },
+];
+
+const TreeMultiselectAsListExample = () => (
+  <Tree
+    defaultSelectedItemIds={['tree-item-122', 'tree-item-123']}
+    selectable
+    aria-label="default"
```

Comment:
This is only improving user experience:
Please if you could label the tree a bit more descriptive. 
I mean more related to the example. 
Something like "Multi select" or "Multi select as a list" or anything what is more related to the example. 
Because currently VoiceOver narrate "default, listbox"

Reply:
Resolved by [1df7b0f](https://github.com/microsoft/fluentui/pull/15147/commits/1df7b0fe7b2c55f88fb05d38f3f816646ed9e583)